src/ui/settings/page.js

```diff
@@ -3,25 +3,26 @@
 import { jsx, Styled } from 'theme-ui';
 
 import { useTranslation } from 'react-i18next';
-import { Container } from '@theme-ui/components';
+import { Box } from '@theme-ui/components';
 
 import LanguageSettings from './language';
 import OpencastSettings from './opencast';
 
 
-const SettingsPage = ({ settingsManager }) => {
+const SettingsPage = ({ settingsManager, hasRecording }) => {
+  console.log(hasRecording);
```

Comment:
console.log...

Reply:
Instead of providing "hasRecording" as a prop you could just ask for the recording context:

```import { useRecordingState } from '../../recording-context';```

and

```  
const { recordings } = useRecordingState();
const hasRecording = recordings.length;
```